### ISSUE TITLE
Feature Request: neural networks from ScikitLearn

### ISSUE THREAD

**Issue by azev77** [alan-turing-institute/MLJModels.jl#199]
MLJ has most models from ScikitLearn, but I believe it's missing neural networks:
https://scikit-learn.org/stable/modules/neural_networks_supervised.html

It would be really awesome to add these models to MLJ.

**Comment by tlienart** [comment]
Yes this was deliberate laziness from my side due to the fact that these models are not very good but I guess an argument in favour of bringing them in is that a simple MLP can be used at a final stage of a learning network which could be useful. 

This would be a reasonably simple PR if you felt up to it since it's pretty much identical to existing code for sklearn models, I'd be happy to help you with it :) otherwise I may do it but probably not soon
